tests/Zend/LocaleTest.php

```diff
@@ -85,6 +85,8 @@ public function tearDown()
             return;
         }
         setlocale(LC_ALL, $this->_locale);
+        Zend_LocaleTestHelper::resetObject();
+        putenv("HTTP_ACCEPT_LANGUAGE");
```

Comment:
This test was actually causing the TranslateTest.php test to fail (since they're both run in the same batch), so doing a little more cleanup here.